todo-with-kuzzle-and-vueJS/README.fr.md

```diff
@@ -0,0 +1,724 @@
+# Todo MVC Step1
+## Pré-requis
+
+Node.js
+
+npm
+
+## Introduction
+Kuzzle permet de gérer de nombreuses données de manière très simple.
+
+Afin de l'installer, vous pouvez vous rendre [ici](https://docs.kuzzle.io/guide/1/essentials/installing-kuzzle/).
+
+Dans ce how-to, nous allons réaliser une simple todo-list utilisant 
+le pattern modèle-vue-controleur avec Kuzzle et VueJS.
+
+Cette première partie mettra en avant les fonctionnalités de 
+Kuzzle telles que:
+- la connexion à Kuzzle,
+- la création d'un index,
+- la vérification de l'existence d'un index,
+- la création d'une collection,
+- la récupération d'une liste des collections existantes,
+- la création d'un document,
+- la suppression d'un document,
+- la mise à jour d'un document,
+- la recherche de documents
+
+Les parties `<template>` et `style`, étant liées à VueJS et Vuetify mais
+n'ayant pas d'interaction avec Kuzzle, ne seront pas détaillées dans ce how-to;
+vous pouvez cependant consulter les fichiers concernés parallèlement à 
+la lecture de ce tutoriel.
+
+## Configuation du projet
+Commençons par installer Vue Cli : 
+```
+npm install -g @vue/cli
+```
+
+Nous pouvons maintenant créer un projet :
+```
+vue create todomvc
+```
+Ajoutez manuellement les fonctionnalités suivantes au projet : `router`
+et `vuex`.
```

Comment:
How do I do that? 

In a documentation, you must absolutely stay consistent. So here you have two options:

1. Guide the reader through all and every steps, even the most simple ones. That's what you did by explaining how to install the Vue CLI.
2. Give only general instructions (and refer to third-party documentations) when the steps are about tooling and prerequisites, and give details only when it's important.

Here, you started by explaining how to install the Vue.js CLI, but then you leave the reader alone with that features installation task.

IMO, I think the second option is better, as your how-to is about kuzzle, not vue.js
So you should propose a link (or links) explaining how to scaffold a vue.js project, and detail only the steps specific to kuzzle

Reply:
I also agree on the second option.